src/transformers/models/qwen3_omni_moe/modular_qwen3_omni_moe.py

```diff
@@ -1196,8 +1197,18 @@ class Qwen3OmniMoeThinkerTextRotaryEmbedding(Qwen3VLMoeTextRotaryEmbedding):
     pass
 
 
+class Qwen3OmniMoeThinkerTextExperts(Qwen3MoeExperts):
+    """
+    ModuleList of experts.
+    """
+
+    def __init__(self, config: Qwen3OmniMoeThinkerConfig):
+        super().__init__(config)
```

Comment:
We can just do the following no?
```suggestion
class Qwen3OmniMoeThinkerTextExperts(Qwen3MoeExperts):
    pass

```